config/rbac/role.yaml

```diff
@@ -163,6 +163,30 @@ rules:
   - get
   - list
   - update
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
should add below too:

```
  - apiGroups:
      - batch
    resources:
      - jobs
    verbs:
      - create
      - watch
      - delete
      - get
      - list
      - update
```

Reply:
https://github.com/streamnative/function-mesh/blob/a1ee4142f9f55accb2d561e36db0bc31eb467502/config/rbac/role.yaml#L44-L54 already have the batch part.

Reply:
oh, ok